pages/linux/quotacheck.md

```diff
@@ -0,0 +1,28 @@
+# quotacheck
+
+> Scan a filesystem for disk usage, create, check and repair quota files.
+> Should only be run by super-user.
+
+- Check all mounted non-NFS filesystems:
+
+`quotacheck --all`
+
+- Force check even if quotas are enabled:
```

Comment:
It forces checking (and writing) of new quota files on file systems. Which can be an issue as the created quota files may be out of sync.

Normally it would be recommended that quota is turned off while the check is run, so to prevent the above issues, quota check tries to remount the file system read-only before starting the scan. After the scan is done it remounts the file system read-write.

Do you think I should add an extra Description line - 
> Scan a file system for disk usage; create, check and repair quota files. 
> It is best to run quota check with quotas turned off to prevent damage or loss to quota files.

notice: I also removed the text about the command being run by super user - I think I should instead append sudo to the start of each example, do you agree?